README.md

```diff
@@ -398,15 +408,16 @@ BatchLoader.for(item).batch(default_value: default_value, cache: cache, key: key
-| `item`          | -                                             | Item which will be collected and used for batching.           |
-| `default_value` | `nil`                                         | Value returned by default after batching.                     |
-| `cache`         | `true`                                        | Set `false` to disable caching between the same executions.   |
-| `key`           | `nil`                                         | Pass custom key to uniquely identify the batch block.         |
-| `items`         | -                                             | List of collected items for batching.                         |
-| `loader`        | -                                             | Lambda which should be called to load values loaded in batch. |
-| `args`          | `{default_value: nil, cache: true, key: nil}` | Arguments passed to the `batch` method.                       |
+| Argument Key      | Default                                       | Description                                                                                                              |
+| ---------------   | --------------------------------------------- | -------------------------------------------------------------                                                            |
+| `item`            | -                                             | Item which will be collected and used for batching.                                                                      |
+| `default_value`   | `nil`                                         | Value returned by default after batching.                                                                                |
+| `cache`           | `true`                                        | Set `false` to disable caching between the same executions.                                                              |
+| `replace_methods` | `nil`                                         | Defaults to the value of `cache`. Set `false` to always use `#method_missing` to respond to methods on the loaded value. |
+| `key`             | `nil`                                         | Pass custom key to uniquely identify the batch block.                                                                    |
+| `items`           | -                                             | List of collected items for batching.                                                                                    |
+| `loader`          | -                                             | Lambda which should be called to load values loaded in batch.                                                            |
+| `args`            | `{default_value: nil, cache: true, key: nil}` | Arguments passed to the `batch` method.                                                                                  |
```

Comment:
could we also pass `replace_methods` to `args` both in the code and the readme for consistency? some may want to be able to read the `replace_methods` value from the block (e.g. if a block was defined and passed separately)

we may also add the new argument to the code example in the API section (the line is quite long, feel free to format if you want):

```rb
BatchLoader.for(item).batch(default_value: default_value, cache: cache, key: key) do |items, loader, args|
  # ...
end
```

Reply:
Done, can you check this looks like you expected?